core/src/main/scala/spinal/core/sim/SimBootstraps.scala

```diff
@@ -274,7 +280,7 @@ object SpinalGhdlBackend {
     if (config.timePrecision != null) {
       vconfig.elaborationFlags = s"--time-resolution=${config.timePrecision.decompose._2}"
     }
-    vconfig.runFlags = config.simulatorFlags.mkString(" ")
+    vconfig.runFlags = config.runFlags.mkString(" ")
```

Comment:
Should we completely squash simulator flags for GHDL?

Reply:
The current implementation squashes both simulatorFlags and runFlags into simulatorFlags.

My implementation keeps those apart and introduces a way to specify runFlags to the simulator.

So my patch does quite the opposite from squashing.

Reply:
I see I missed the line above this that sets elaboration flags from sim flags